sdk/tests/deqp/functional/gles3/es3fTextureSpecificationTests.js

```diff
@@ -2639,8 +2639,7 @@ goog.scope(function() {
                 format: this.m_texFormat,
                 width: this.m_width,
                 height: this.m_height,
-                data: data,
-                offset: this.m_texFormat.getPixelSize() * this.m_width * this.m_height
+                data: data
             });
```

Comment:
c++ code for comparison: https://code.google.com/p/chromium/codesearch#chromium/src/third_party/deqp/src/modules/gles3/functional/es3fTextureSpecificationTests.cpp&l=1656